build/Dockerfile

```diff
@@ -1,14 +1,16 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
-ENV OPERATOR_PATH=/go/src/github.com/openshift/splunk-forwarder-operator
-COPY . ${OPERATOR_PATH}
-WORKDIR ${OPERATOR_PATH}
+#ENV OPERATOR_PATH=/go/src/github.com/openshift/splunk-forwarder-operator
+#COPY . ${OPERATOR_PATH}
```

Comment:
I'd like remove them instead of comment out

Reply:
done.